std_msgs/Header header
std_msgs/String object_class
std_msgs/Float32[] softmaxes
std_msgs/Float32[] logits
std_msgs/Bool is_valid